cmd/worker/main.go

```
@@ -83,7 +83,7 @@ func HandleKeyGeneration(ctx context.Context, t *asynq.Task) error {
 		ECDSAPublicKey: keyECDSA,
 	}
 
-	resultBytes, err := json.Marshal(result)
+	resultBytes, _ := json.Marshal(result)
```

Could you please not ignore the error? handle it appropriately